source/_integrations/habitica.markdown

```diff
@@ -145,3 +145,10 @@ Also an event `habitica_api_call_success` will be fired with the following data:
     "id": "NEW_TASK_UUID"}
 }
 ```
+
+This Markdown will list the tasks saved in `sensor.habitica_USER_todos.attributes.items` in a card.
+
+```markdown
```

Comment:
This isn't Markdown, by the Jinja2 template syntax. Use `jinja` as a language flag instead.